iphone/Maps/UI/Settings/MWMSettingsViewController.mm

```diff
@@ -76,10 +84,12 @@ - (void)configCommonSection {
   {
     self.is3dCell.isEnabled = false;
     [self.is3dCell configWithDelegate:self title:L(@"pref_map_3d_buildings_title") isOn:false];
+    [self.is3dCell addGestureRecognizer:singleFingerTap];
```

Comment:
There is nothing wrong with this code, but it can likely be improved:
1. Gesture recogniser can be created only here, when necessary.
2. It can be removed without keeping its pointer, by accessing this property (clearing it?): https://developer.apple.com/documentation/uikit/uiview/1622542-gesturerecognizers

Reply:
Removed calls to `addGestureRecognizer:` and `removeGestureRecognizer`. No need to store pointer in a field.